.github/container/Dockerfile.maxtext.amd64

```diff
@@ -30,5 +40,6 @@ ADD test-maxtext.sh /usr/local/bin
 FROM mealkit as final
 
 RUN pip-finalize.sh
+RUN pip install typing-extensions==4.10.0
```

Comment:
It is odd that you need to do it as it defeats the purpose of using `pip-compile`. Can you elaborate the conflict?

Also, if this strict pinning is truly the version you need, then it should go in `/opt/pip-tools.d/requirements-maxtext.in`. If for some reason it causes a conflict because you put it in there, we should resolve that

Reply:
@terrykong I removed the `typing-extensions` pin. It should be fine with the current container config

Reply:
Just commenting about the resolution: the typing-extensions pin wasn't needed b/c pip-compile found the right version. @kocchop found that it was needed in building maxtext in a non-jax-toolbox dockerfile